pkg/model/errors.go

```diff
@@ -38,12 +38,13 @@ var (
 	ErrValidationFailed         = errors.New("validation failed")
 	ErrInvalidFormat            = errors.New("invalid format")     // entire request body has invalid format
 	ErrInsufficientFunds        = errors.New("insufficient funds") // insufficient funds on some address in a blockchain network
+	ErrInvalidJSON              = errors.New("invalid JSON")
```

Comment:
Is this error type still used in the code?

Reply:
It is:

```
[13:03] optriment/optrispace-backend ‹refactor/extend-validations*›$ grep -r ErrInvalidJSON .
./pkg/web/handlers.go:                          cause = model.ErrInvalidJSON // "invalid JSON"
./pkg/model/errors.go:  ErrInvalidJSON              = errors.New("invalid JSON")
```